user-service/src/controllers/userController.ts

```diff
@@ -107,6 +107,23 @@ export const updateUser = async (req: Request, res: Response) => {
 			return res.status(400).json({ message: "Please provide both current password and new password" });
 		}
 
+    // sanity check
+    // Valid: user@example.com, invalid: user@ example.com
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+		if (!emailRegex.test(email)) {
+			return res.status(400).json({ message: "Invalid email format" });
+		}
+
+    const existingUser = await User.findOne({ username });
```

Comment:
Pls help to fix the spacing.

Reply:
lmao

Reply:
just fix the rest of the changes in a refactoring PR else the changes will be messy

but i'll fix this cos i added it